configs/tester/tester.toml

```diff
@@ -36,6 +36,19 @@ group-name = "model-store"
 model-mount-destination = "/models"
 model-definition-path = "./model-definition.yaml"
 
+[context.model-service-scale-replicas]
+replicas = 2
```

Comment:
It's confusing because there are two replicas defined in the configuration file.
Let's change one side to initial_replica to make it explicit.